demisto_sdk/commands/common/hook_validations/pack_unique_files.py

```diff
@@ -41,6 +41,10 @@ def _add_error(self, error, file_path):
         """Adds error entry to a list under pack's name
         Returns True if added and false otherwise"""
         error_message, error_code = error
+
+        if '/' not in file_path:
```

Comment:
not sure that this is a good check, it will not work on windows

Reply:
ok have another idea